commands/commands.go

```diff
@@ -55,8 +60,13 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 	log.Debugf("%s.Run start", c.Name)
 
 	cmd := ArgsToCmd(c.Exec, c.Args)
```

Comment:
On second look, I'm honestly not sure why that function exists at all. The function it wraps is variadic, so we can just call it directly and safely pass in the zero-value for `Command.Args`. Maybe there was more logic there at some point in the past... I don't feel like code-spelunking over it now.

Just pushed a commit removing `ArgsToCmd` entirely and just calling `os/exec.Command` instead.